docs/image-redactor/index.md

```diff
@@ -1,22 +1,113 @@
 # Presidio Image Redactor
 
+***Please notice, this package is still in beta and not production ready.***
+
 ## Description
 
-## Table of contents
+The Presidio Image Redactor is a Python based module for detecting and redacting PII
+text entities in images.
+![img.png](../assets/image-redactor-design.png)
 
 ## Installation
 
-## Getting started
+Pre-requisites:
+
+- Install [Tesseract OCR](https://github.com/tesseract-ocr/tesseract#installing-tesseract) by following the
+  instructions on how to install it for your operating system.
+
+  The Image-Redactor was tested with a minimal version of 4.0.0
 
-- Running in Python.
-- Running using Docker.
-- Customizing.
-- API reference.
+=== "Using pip"
+    
+    !!! note "Note"
```

Comment:
note "note" ?

Reply:
This is the syntax for a mkdocs extention, admonition, better formatting notes:
![image](https://user-images.githubusercontent.com/15013757/109482830-2b444480-7a87-11eb-847c-efe70cea6649.png)
Downside is a 'dirtier' markdown